db/migrate/20150331144451_changetablenames.rb

```diff
@@ -0,0 +1,7 @@
+class Changetablenames < ActiveRecord::Migration
+  def change
+    rename_table :trucks, :vendors
+    rename_table :truck_categories, :vendor_categories
+    rename_table :truck_location_schedules, :vendor_location_schedules
+  end
+end
```

Comment:
You shouldn't change this filename, but in the future use underscores to separate words  when generating migrations or naming files!